browser/js/calculator.js

```diff
@@ -0,0 +1,265 @@
+( function() {
+
+  function Calculator( domElement ) {
+    this.dom = domElement,
+    this.data = {
+      memory: '0',
+      current: '0',
+      operation: '',
+      result: '0',
+      MAX_LENGTH: 11
+    },
+    this.onButtonClick = this.onButtonClick.bind( this ),
+    this.onKeyPress = this.onKeyPress.bind( this ),
+    this.addListeners()
+  }
```

Comment:
I see that the function closes here, It would have been great to add a space in between the closing and opening brackets. Try to do that for all just so it could be easier to read, I got confused and was having a hard time reading whether the whole thing was part of the structure or not.